content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -84,6 +84,15 @@ document better and be able to customize based on your needs.
     kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com
     ```
 
+## Upgrade
+1. Make sure kubectl talks to your cluster.
```

Comment:
We should tell them how to do this. (Even though we tell them higher up on the page, people may come directly to this section, and then not know what to do.) I think you can copy lines 69-72.